src/components/Articles/ArticleCard.tsx

```diff
@@ -20,18 +20,18 @@ export default function PostCard({ post }: PostCardProps) {
     favoritesCount,
     tagList,
     slug,
-  } = post;
+  } = article;
   return (
-    <div className={styles.postCard}>
+    <div className={styles.articleCard}>
       <div className={styles.articleMeta}>
-        <Link href={`${ROUTES.userArticlePage}${author.username}`}>
-          <div className={styles.postInfo}>
+        <Link href={`${ROUTES.USERARTICLEPAGE(author.username).href}`}>
```

Comment:
상수를 만들때 단어가 여러개인 경우에는 단어별로 _(언더스코어) 구분해주는게 가독성에 훨씬 좋을 것 같아요~
```suggestion
        <Link href={`${ROUTES.USER_ARTICLE_PAGE(author.username).href}`}>
```